app/src/main/java/com/hover/stax/faq/FAQ.kt

```diff
@@ -0,0 +1,40 @@
+package com.hover.stax.faq
+
+import android.app.Application
+import androidx.lifecycle.AndroidViewModel
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import com.google.firebase.firestore.ktx.firestore
+import com.google.firebase.firestore.ktx.firestoreSettings
+import com.google.firebase.ktx.Firebase
+import timber.log.Timber
+
+data class FAQ (var id: String, var topic:String, var content:String)
+
+internal fun getFAQList() : LiveData<List<FAQ>> {
+    val db = Firebase.firestore
+    val settings = firestoreSettings { isPersistenceEnabled = true }
+    db.firestoreSettings = settings
+
+    val liveData : MutableLiveData<List<FAQ>> = MutableLiveData()
+
+    db.collection("faqs")
+            .get().addOnSuccessListener { result ->
+                val faqList : MutableList<FAQ> = mutableListOf()
+                for (document in result) {
+                    faqList.add(FAQ(document.id, document.data["topic"].toString(), document.data["content"].toString()))
+                }
+                
+                liveData.value = faqList
+                return@addOnSuccessListener
+            }.addOnFailureListener { exception ->
+                Timber.e(exception)
+                return@addOnFailureListener
+            }
+
+    return liveData
+}
+
+class FaqViewModel(application: Application) : AndroidViewModel(application) {
```

Comment:
Since we're not using the application instance, this can be simplified to use the `ViewModel` class with no params.